flytekit/models/annotation.py

```diff
@@ -42,6 +42,19 @@ def from_flyte_idl(cls, proto):
 
         return cls(annotations=_json_format.MessageToDict(proto.annotations))
 
+    @classmethod
+    def merge_annotations(cls, annotation: "TypeAnnotation", other_annotation: "TypeAnnotation") -> "TypeAnnotation":
```

Comment:
let's move this out of the model in the future... keep the models as easy to delete as possible.

Reply:
good call.